schemas/commonTexture.schema.json

```diff
@@ -44,9 +44,9 @@
     },
     "size": {
       "type": "integer",
-      "maximum": 4194304,
+      "maximum": 10485760,
       "errorMessage": {
-        "maximum": "Texture map ${1/name} exceeds maximum allowed storage size of 4 MB."
+        "maximum": "Texture map ${1/name} exceeds maximum allowed storage size of 10 MB."
```

Comment:
Bumped this after testing with the PUBG asset locally, and it failing this check

Reply:
For next-gen studio yes, so people can upload. However it won't impact any legacy frontends right? Since I've increase the limit rather than decreased it, so it shouldn't negatively impact any existing limits in place in any frontends.